extensions/packageManager/src/node/index.ts

```diff
@@ -55,9 +134,33 @@ export default async (composer: IExtensionRegistration): Promise<void> => {
   };
 
   const LibraryController = {
+    getReadme: async (req: any, res: any) => {
+      try {
+        const moduleName = req.params.packageName;
+        if (!moduleName) {
+          res.status(400).json({
+            message: 'missing module name on request',
+          });
+        } else {
+          const moduleURL = 'https://registry.npmjs.org/' + moduleName;
+          const raw = await axios.get(moduleURL);
+          res.status(200).json(raw.data);
+        }
+      } catch (error) {
+        res.status(400).json({
```

Comment:
Do we have http response information in error? If there is a status code it would be better to pick it up and send that as status instead of always 400. If we don't have that info, 400 is ok.

Reply:
I have made the remote response value available where possible! good one.